src/components/cards/StatusCard.tsx

```diff
@@ -0,0 +1,41 @@
+import cn from "clsx";
+import React from "react";
+import { Status } from "@/types";
+import { Badge } from "../badge/Badge";
+import { Icons } from "@instill-ai/design-system";
+
+export type StatusProps = Status;
+
+const getIcon = (change: number, statusname: string) => {
```

Comment:
@iamnamananand996 We have discuss this before, the Status is guarded by our API clearly. So we should not use string but use the Status type provided by the backend directly!